scripts/generate-deployment-yamls.sh

```diff
@@ -9,6 +9,9 @@ if [[ -z $VERSION ]]; then
     exit 1
 fi
 
+cat chart/Chart.yaml | sed -e "s/version: .*/version: $VERSION/" > chart/Chart.yaml.new && mv chart/Chart.yaml{.new,}
```

Comment:
I think we should use `release-please` for this in the future, it can update versions in any file.